publish.sh

```diff
@@ -68,10 +68,11 @@ function pre_process {
   cd ..
   if [ $version == 'main' ]; then
     output="\ \ \ \ git:\n      url: https://github.com/flame-engine/flame.git\n      ref: main\n      path: packages/flame"
-    find . -name "*.md" -exec sed -i "/<VERSION>.*/a $output" {} \;
-    find . -name "*.md" -exec sed -i "s/ <VERSION>//" {} \;
+    find . -name "*.md" -exec sed -i "/--VERSION--.*/a $output" {} \;
+    find . -name "*.md" -exec sed -i "s/ --VERSION--//" {} \;
+    find . -name "resources.md" -exec sed -i "s/--VERSION--/latest/" {} \;
```

Comment:
This line is not needed
```suggestion
```

Reply:
Aah, or maybe it is, didn't see that this was for the main branch.
But this needs to be done before the other replacements, otherwise is already going to be replaced in the other sed command.